### PR TITLE
Minimum environment test

### PR DESCRIPTION
**Pull request by Hakuyume**
I add tests for minimum environments, which do not contain `cv2`, `scikit-learn`, and `matplotlib`.

### CODE REVIEW

File: environment_minimum.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: chainercv
+channels:
+- !!python/unicode
+  'menpo'
```
Comment (yuyu2172) [inline environment_minimum.yml]: Line 3 and 4 are not necessary.

File: environment_minimum.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: chainercv
```
Comment (yuyu2172) [inline environment_minimum.yml]: Change name of the environment to `chainercv_minimum`?

File: tests/extensions_tests/detection_tests/test_detection_vis_report.py [hunk 3]
```diff
@@ -11,6 +11,13 @@
 from chainercv.utils import ConstantReturnModel
 from chainercv.utils import DummyDataset
 
+import importlib
+try:
+    importlib.import_module('matplotlib')
```
Comment (yuyu2172) [inline tests/extensions_tests/detection_tests/test_detection_vis_report.py]: Why do you use `importlib`?
I am just curious.

File: tests/extensions_tests/semantic_segmentation_tests/test_semantic_segmentation_vis_report.py [hunk 4]
```diff
@@ -11,6 +11,13 @@
 from chainercv.utils import ConstantReturnModel
 from chainercv.utils import DummyDataset
 
+import importlib
+try:
+    importlib.import_module('matplotlib')
```
Comment (yuyu2172) [inline tests/extensions_tests/semantic_segmentation_tests/test_semantic_segmentation_vis_report.py]: ditto

**Comment by yuyu2172** [comment]
Great PR!
Thanks!

File: tests/extensions_tests/detection_tests/test_detection_vis_report.py (hunk 3 above)
Comment (Hakuyume) [inline tests/extensions_tests/detection_tests/test_detection_vis_report.py]: When I used `import`, the flake8 test failed because imported `matplotlib` was unused.

Comment (yuyu2172) [inline tests/extensions_tests/detection_tests/test_detection_vis_report.py]: You can add `#NOQA`.
I think not using `importlib` is simpler.

Comment (Hakuyume) [inline tests/extensions_tests/detection_tests/test_detection_vis_report.py]: Thank you! I will use `#NOQA`.

File: environment_minimum.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+name: chainercv_minimum
+channels:
+- !!python/unicode
+  'defaults'
+dependencies:
+  - numpy
+  - pip:
+    - chainer
```
Comment (yuyu2172) [inline environment_minimum.yml]: Please add pillow.

**Comment by yuyu2172** [comment]
I am sorry, but could you fix another point?
Pillow is included in the minimum requirements of ChainerCV, even though it is not in Chainer.

**Comment by yuyu2172** [comment]
LGTM!
